pages/evaluation.md

```diff
@@ -1,35 +1,47 @@
 # Butterfly Hybrid Detection: Evaluation
-This challenge is an anomaly detection problem. The goal is to find hybrid butterfly instances.
+This challenge is an anomaly detection problem. The goal is to find hybrid butterfly instances. There are two sets of images considered in the test set, as described below.
 
-Specifically, the test data comprise:
-- A collection butterfly species' (denoted as Species A) subspecies. Participants can treat these subspecies as different object classes.
-- Hybrids of these subspecies: these cases are produced by parents from different subspecies. Participants can treat hybrids with different combinations of parent subspecies as different object classes. We call the hybrid which is most common in the dataset the "signal" hybrid; it is the only hybrid provided for training.
-- The goal of the challenge is to distinguish between non-hybrid instances and hybrid instances.
+## The First Component of the Test Dataset
+This test data set comprises:
+- An image collection of butterfly **Species A subspecies**. These are non-hybrid instances.
+- An image collection of **Species A hybrids**, which are from parents from different subspecies of Species A. Note that in the training data, we only provide one type of hybrid (the "signal" hybrid), from one specific combination of parents. In the test data, there are also "non-signal" hybrids, which are from other combinations of parents. 
 
-Moreover, Species A has a mimic (or co-mimic) Species B. These two butterfly species have quite similar appearances to evade predators. The test data thus further contains:
-- A mimic butterfly species' (denoted as Species B) subspecies. Specifically, we only consider the two subspecies that correspond to the parent subspecies of Species A's signal hybrid.
-- Hybrids of these two subspecies.
-- The aim of this test set is to see whether an anomaly detection algorithm for butterflies is transferrable across mimic butterflies.
+The goal of the challenge is to distinguish between non-hybrid instances and hybrid instances.
```

Comment:
```suggestion
The goal of the challenge is to elicit new ways of solving this problem through designing a model to distinguish between non-hybrid instances and hybrid instances.
```
Would this work? Though this might be more appropriate here as a task since it's just the first component part?

Reply:
I think this would work.